app/views/pieces/show.html.erb

```diff
@@ -1,19 +1,22 @@
-
-
-
-
-
 <h1 class="text-center">The Pawnshop</h1>
 
-  
+  <div id="turn">
+      <% if current_game.white_player_turn == true %>
```

Comment:
we could refactor it to one line saying: 

`<p>The current turn belongs to: current_game.player_at_turn %></p>`

And on the game model you can define the method `player_at_turn` (maybe you can find a better name 😄 ) and move the logic over there...

Reply:
Ok I'll look at cleaning this up thank you.